### PR TITLE
Overview css

### PR DESCRIPTION
**Pull request by chiakat**
@afong888 Can you review my css updates for Overview?

### CODE REVIEW

**Comment by afong888** [comment]
Picking Up

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -48,7 +46,7 @@ const App = () => {
       </div>
       <div><RelatedItems productId={currentId} /></div>
       <div className="questions-container"><QuestionsAndAnswers productId={currentId} /></div>
-      <div><ReviewList /></div>
+      <div id="reviews"><ReviewList /></div>
```
Comment (afong888) [inline client/src/components/App.jsx:49]: Why did you add id reviews to reviewList for kevin's component?

Comment (chiakat) [reply client/src/components/App.jsx:49]: @afong888 My Star Ratings review link should point to the Reviews & Ratings section so I added an id so that my link could find it. The link is in ProductInfo.jsx line 49.
FYI @kyang9021
